src/makeInfinityAdder.js

```diff
@@ -3,8 +3,26 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(num) {
```

Comment:
The initial implementation of `makeInfinityAdder` accepts a parameter `num`, which contradicts the task description. The `makeInfinityAdder` function should not accept any arguments, as it is supposed to return a function that will handle the addition.